spec.html

```diff
@@ -562,7 +562,7 @@ <h1><ins>GetCycleRoot ( _module_ )</ins></h1>
     <emu-clause id="sec-asyncmodulexecutionfulfilled" aoid="AsyncModuleExecutionFulfilled">
       <h1><ins>AsyncModuleExecutionFulfilled ( _module_ )</ins></h1>
       <emu-alg>
-        1. Assert: _module_.[[Status]] is `"evaluated"`.
+        1. Assert: _module_.[[Status]] is `"evaluating-async"`.
```

Comment:
Can you please double-check that it can't be "evaluated"? If so, LGTM.

Reply:
Rejection can come from a dependency rejection or an execution rejection. Fulfillment will from async execution, which is only started after all dependencies have completed successfully (and didn't reject). So yes execution fulfillment will never happen after a former rejection or fulfillment. And all modules that are waiting for any type of async fulfillment will be in "evaluating-async" after InnerModuleEvaluation has completed.

Reply:
Okay, thanks.